### ISSUE TITLE
Gear Icon for "Favorites" feels misleading

### ISSUE THREAD

**Issue by robinsoner** [dgventura/gambl#5]
The Gear Icon is generally used for "Settings" or "Configuration".  Perhaps a ♥ icon instead?  Hover text, too, would be helpful.

Case in point: I didn't know that the favorite button did what it did until the app crashed and revealed the function names in the call stack.

**Comment by dgventura** [comment]
Yeah, I just picked the first icon in the palette I had.  A star or heart would be better, as well as hover text.  Thanks.
